exporter/opentelemetry-exporter-prometheus-remote-write/pyproject.toml

```
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+[project]
+name = "opentelemetry-exporter-prometheus-remote-write"
+dynamic = ["version"]
+description = "Prometheus Remote Write Metrics Exporter for OpenTelemetry"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+  "protobuf ~= 4.21",
+  "requests ~= 2.28",
+  "opentelemetry-api ~= 1.13.0",
+  "opentelemetry-sdk ~= 1.13.0",
```

I believe `~= 1.13.0` is equivalent to `>= 1.13.0, == 1.13.0`. But it should be compatible with other minor versions right? Going off the template:

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/9beb6b29b08a0c5333cb01139134eab03bbc11c8/_template/pyproject.toml#L31

does this work?

```suggestion
  "opentelemetry-api ~= 1.12",
  "opentelemetry-sdk ~= 1.12",
```